pyuvsim/tests/test_uvsim.py

```diff
@@ -521,10 +521,14 @@ def test_uvdata_init():
                        'Based on UVData file: ' + EW_uvfits_file + '. Npus = 1.'
                        + hera_uv.pyuvdata_version_str)
     hera_uv.instrument = hera_uv.telescope_name
-    nt.assert_equal(hera_uv._antenna_positions, uvdata_out._antenna_positions)
+    nt.assert_true(np.allclose(hera_uv.antenna_positions, uvdata_out.antenna_positions))
```

Comment:
I noticed it was written this way and wasn't sure why. I suppose comparing the raw UVParameters automatically does the np.isclose() test instead of exact equality? I figured this would be more explicit.

I can't get any tests to break by changing to center=False... I noticed it made a difference by comparing reference sims. I thought at least the `test_mpi_uvsim` tests would fail, since those compare a simulated UVData object to a saved uvfits file, but even those passed.

Reply:
Yes, you're right, using the UVParameters does use allclose and it also uses the tolerance associated with the antenna_positions parameter, rather than the allclose default, which I think is what you're using here.

I think I'm still super confused about the problem. If you look at these two antenna position values are they identical?